Lombiq.Tests.UI.Shortcuts/Startup.cs

```diff
@@ -11,5 +12,8 @@ public override void ConfigureServices(IServiceCollection services)
     {
         services.AddSingleton<IInteractiveModeStatusAccessor, InteractiveModeStatusAccessor>();
         services.AddAsyncResultFilter<ApplicationInfoInjectingFilter>();
+
+        // To ensure we don't encounter any concurrency issue, enable EnableThreadSafetyChecks for all tests.
+        services.Configure<YesSqlOptions>(options => options.EnableThreadSafetyChecks = true);
```

Comment:
I didn't know this configuration. Good idea.

Reply:
It's a new config [added recently](https://github.com/OrchardCMS/OrchardCore/pull/15961).